thrift/handler.cc

```diff
@@ -767,7 +767,7 @@ class thrift_handler : public CassandraCobSvIf {
     }
 
     void describe_partitioner(thrift_fn::function<void(std::string const& _return)> cob) {
-        cob(dht::global_partitioner().name());
+        cob(_db.local().get_config().partitioner());
```

Comment:
I'm confused why this is better... I'm worried that it is worse: The partitioner is still "global" in the sense we have just one _db, and if later we want to have a per-table partitioner, this code will still be broken - but will now be harder to find because it no longer has that easy-to-find global_partitioner() call.

This concern also applies to several other changes in other patches.

Reply:
I'm actually having a following PR that does the switch to per table partitioner. This PR is a prelude for that. 

To address your concern. Even with per table partitioner there will be a default partitioner and this method will return the name of that default partitioner. The default partitioner will be the one used when a table won't have it's own partitioner set.

We will have to extend the protocol to inform drivers about partitioners used by different tables but that's another story.

Bottom line is, if you're worried this PR will make stuff worse for per table partitioners - stop. It's part of the effort to make it happen.